cmd/aci.go

```diff
@@ -78,13 +77,15 @@ func runACIHandler(cmd *cobra.Command, args []string) {
 	cmd.Flags().Int("aci-sleep-time", 100, "Time in ms, to sleep between HTTP POST to AM. (default: 100)")
 	cmd.Flags().Int("aci-send-time", 60000, "Time in ms, to complete HTTP POST to AM. (default: 60000)")
 	cmd.Flags().Bool("aci-enable-consul", false, "Whether to use consul for host discovery (default: false)")
+	cmd.Flags().String("consul-aci-group-host", "", "Consul group host used on filtering the host list (empty by default: matches everything)")
 	cmd.Flags().Bool("aci-debug", false, "Enable debug level logs for acigo. (default: false)")
 
 	checkRequiredFlags(cmd, args, []string{})
 
 	config := ACIConfig(cmd)
 
-	client, err := acigo.NewACIClient(of.ACIClientConfig{Hosts: []string{config.SourceHostname},
+	// Hosts are ignored on the initialization, they are set on runtime on every step for the cycle of pulling
+	client, err := acigo.NewACIClient(of.ACIClientConfig{Hosts: []string{"ignored"},
```

Comment:
```suggestion
	client, err := acigo.NewACIClient(of.ACIClientConfig{
```
You don't need to initialize.